tests/Tests/600__ai_apps/600__anaconda_commercial_edition/600__anaconda_commercial_edition.robot

```diff
@@ -74,11 +74,14 @@ Verify User Is Able to Activate Anaconda Professional
   Should Be Equal  ${val_result[0]}  ${VAL_SUCCESS_MSG}
   Wait Until Keyword Succeeds    1200  1  Check Anaconda CE Image Build Status  Complete
   Go To  ${ODH_DASHBOARD_URL}
+  Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
   Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
   Launch JupyterHub Spawner From Dashboard
   Run Keyword And Continue On Failure  Verify Anaconda Element Present Based On Version
   Run Keyword And Continue On Failure  Verify Anaconda Element Enabled Based On Version
   Spawn Notebook With Arguments  image=s2i-minimal-notebook-anaconda
+  Verify Git Plugin
+  Capture Page Screenshot
```

Comment:
I haven't included it. It's for checking spawning an image. I've just added the verify git plugin in between of them so that both the thing get's checked.

Reply:
hmm I don't see the `Capture Page Screenshot` in the current version of the code

Reply:
Please, can you remove this screenshot here?